float64[]   ini_pose

---

float64[]   trans
float64[]   pos
float64[]   euler